packages/components/src/components/dropdown/items.jsx

```diff
@@ -1,8 +1,7 @@
 import classNames          from 'classnames';
 import PropTypes           from 'prop-types';
 import React               from 'react';
-import { Popover }         from 'App/Components/Elements/Popover';
-import { getCurrencyName } from '_common/base/currency_base';
+import Popover             from 'Components/dummy.jsx';
```

Comment:
None of the dropdown items are using the `Popover` now?

Reply:
Since Popover is not yet ported, I did use a dummy element.

Reply:
@reza-fs yes won't that break existing dropdowns with `Popover`?

Reply:
:thinking: Yes it does. allow me to hot patch this one too.